eva_submission/eload_ingestion.py

```diff
@@ -0,0 +1,111 @@
+import string
+import subprocess
+
+from bs4 import BeautifulSoup
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.mongo_utils import get_mongo_connection_handle
+import requests
+
+from eva_submission.eload_submission import Eload
+
+
+def sanitize(s):
+    """Lowercases and removes punctuation from s, e.g. to make it a legal MongoDB identifier."""
+    return s.lower().translate(str.maketrans('', '', string.punctuation))
+
+
+class EloadIngestion(Eload):
+    config_section = 'ingestion'  # top-level config key
+    all_tasks = ['metadata_load', 'accession', 'variant_load']
+
+    def __init__(self, eload_number):
+        super().__init__(eload_number)
+        self.eload_cfg.set(self.config_section, 'ingestion_date', value=self.now)
+
+    def ingest(self, db_name=None, tasks=None):
+        # TODO assembly/taxonomy insertion script should be incorporated here
+        self.check_variant_db(db_name)
+
+        if not tasks:
+            tasks = self.all_tasks
+
+        if 'metadata_load' in tasks:
+            self.load_from_ena()
+        if 'accession' in tasks:
+            self.warning('Accessioning not yet supported, skipping.')
+        if 'variant_load' in tasks:
+            self.warning('Variant loading not yet supported, skipping.')
+
+    def get_db_name(self):
```

Comment:
While we haven't stuck to [any strict rules for the database name in the past](https://github.com/EBIvariation/eva-submission/pull/12/files#r563308997), we might have to agree upon some convention now that we are automating this. @tcezard @apriltuesday Let's discuss this more during one of the stand-ups.

Reply:
As discussed we'll postpone this discussion to another time.